setting_up_firely_server/configuration/logsettings.rst

```diff
@@ -266,6 +266,26 @@ Firely Server can also log to AWS Cloudwatch. What you need to do:
 			},
 		],
 
+Splunk
+^^^^^^
+Firely Server can also log to Splunk. What you need to do:
+
+#. Setup a Splunk environment as described by the Splunk documentation
+#. Create a ``HTTP Event Collector`` for the application, save the ``Token Value`` for later use
+#. Check in the ``Global Settings`` in the ``HTTP Event Collector`` screen which port is used
+#. Add the correct sink to the logsettings.json::
```

Comment:
```suggestion
#. Create an ``HTTP Event Collector`` for Firely Server and record the ``Token Value`` to be used below
#. Refer to the ``Global Settings`` section in the ``HTTP Event Collector`` interface to identify the assign port
#. Add the sink values to the logsettings.json::
```

Reply:
I realized after doing this that the other doc looks much like the way you wrote it. Feel free to edit or ignore. We can refine all of it another day.